### PR TITLE
#4 Add github workflow for Heroku - Berkeli

### PR DESCRIPTION
**Pull request by berkeli**
## Description

Added github workflow that will spin up QA environment from the branch of the pull request when a pull request is created

## Related to

[Set up Heroku #4](https://github.com/berkeli/ldn8-final-project/issues/4)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have carefully reviewed my own code
- [x] I have commented my code
- [ ] ~I have updated any documentation~

### CODE REVIEW

File: .github/workflows/qa-env-on-pull-request.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+name: Create QA environment when a pull request is created against master and
```
Comment (pgonzazalez) [inline .github/workflows/qa-env-on-pull-request.yml]: master and?

File: .github/workflows/qa-env-on-pull-request.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: ${{ github.event.action == 'closed' && 1 || 0 }}
+          ref: ${{ github.event.action != 'closed' && github.head_ref || '' }}
+      - name: Login to Heroku
+        uses: akhileshns/heroku-deploy@v3.12.12
+        with:
+          heroku_api_key: ${{ secrets.HEROKU_API_KEY }}
+          heroku_email: user@example.com
```
Comment (pgonzazalez) [inline .github/workflows/qa-env-on-pull-request.yml]: don't you want to change this bit?

**Review by pgonzazalez — COMMENTED** [review]
Just small things. :)

File: .github/workflows/qa-env-on-pull-request.yml (hunk 1 above)
Comment (berkeli) [reply .github/workflows/qa-env-on-pull-request.yml]: thanks :)

File: .github/workflows/qa-env-on-pull-request.yml (hunk 2 above)
Comment (berkeli) [reply .github/workflows/qa-env-on-pull-request.yml]: thanks!

**Review by pgonzazalez — COMMENTED** [review]
LGTM. 🚀 
I will leave approving the PR to Delroy and Irina.

**Review by IrinShilova — APPROVED** [review]

**Pull request merged**